### DISCUSSION TITLE
Is it possible to generate grad-cams using onnx?

### DISCUSSION THREAD

**Discussion by lus105** [onnx/onnx#4945] [Q&A]
For my grad-cams generation I am using trained squeezenet model from https://pytorch.org/hub/pytorch_vision_squeezenet/. For heatmap generation I am using this repo https://github.com/jacobgil/pytorch-grad-cam and it works flawlessly. The layer of interest is 'features_12_expand3x3_activation'. I am wondering if it is possible to convert a model to onnx format and use it for grad-cam generation?

**Comment by pawara101** [comment]
I have faced similar situation when deploying the ONNX exported model and using the model for GradCAM heat map creation. Is there a way to use ONNX model to use the model parameters ?
